sourcefiles/base/basepatch.py

```diff
@@ -359,6 +363,103 @@ def apply_misc_patches(ct_rom: ctrom.CTRom):
     ct_rom.rom_data.write(b'\xEA\xEA')  # Overwrites ORA $71
 
 
+def patch_tabs(ct_rom: ctrom.CTRom):
```

Comment:
Does this play nicely with the fast tabs flag?

Reply:
I'll need to test it.  Haven't run that use case yet.  I'll let you know

Reply:
Did a quick check of tabs that are available at the beginning of the seed.  Fast tabs are still fast